src/test/osd/TestOSDMap.cc

```diff
@@ -46,6 +46,7 @@ class OSDMapTest : public testing::Test,
   static const string range_addrs[];
   static const string ip_addrs[];
   static const string unblocked_ip_addrs[];
+  static constexpr string EC_RULE_NAME = "erasure";
```

Comment:
@JoshSalomon the make check failure is related to this line:

```
/home/jenkins-build/build/workspace/ceph-pull-requests/src/test/osd/TestOSDMap.cc:49:27: error: constexpr variable cannot have non-literal type 'const std::string' (aka 'const basic_string<char>')
  static constexpr string EC_RULE_NAME = "erasure";
                          ^
/usr/bin/../lib/gcc/x86_64-linux-gnu/11/../../../../include/c++/11/bits/basic_string.h:85:11: note: 'basic_string<char>' is not literal because it is not an aggregate and has no constexpr constructors other than copy or move constructors
    class basic_string
          ^
1 error generated.
```